### DISCUSSION TITLE
Golang SDK - Live Transcription multiple calls

### DISCUSSION THREAD

**Discussion by iliesh** [deepgram/community#829] [General help] [answered]
I have an application will receive calls from the PBX, and then send RTP packets to Deepgram servers using the Go SDK. Everything works okay when processing a single call.
My question is: How should I proceed to process multiple calls simultaneously? At present, I create a Deepgram client with a specific callback when the application starts and use it to send RTP traffic to the DG servers.
To process multiple calls, should I create a new instance of the Deepgram client for each call? Are there any limits to the number of clients that can be created? Could I process one hundred calls simultaneously? Or is
there an option to use a single client and mark each UDP packet sent to DG with a specific tag to distinguish the call in the callback function?

Thank you

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by SandraRodgers** [comment, marked as answer]
The tag option could be used to help distinguish between different calls. The tag feature in Deepgram is primarily intended for tracking and filtering usage. You can apply multiple tags or multiple extra key-value pairs by submitting the query parameter multiple times in your API request.

See documentation at https://developers.deepgram.com/docs/tagging
